js/modules/k6/k6.go

```diff
@@ -158,6 +159,9 @@ func (mi *K6) Check(arg0, checks goja.Value, extras ...goja.Value) (bool, error)
 	if state == nil {
 		return false, ErrCheckInInitContext
 	}
+	if checks == nil {
+		return false, fmt.Errorf("No checks provided to `check`")
```

Comment:
It seems like there is nothing that we can format :wink:

Reply:
fixed in [3be0fd4](https://github.com/grafana/k6/pull/2387/commits/3be0fd4d976396490f870ae7de1f87d760b87296)